database/papi_template.py

```diff
@@ -7,384 +7,384 @@
-				b'H/8cDSzOuCs4rVqiYUIghSJACIkJUQMXL6J2Rx0AQJvhnhSgm9zHSBwuJpfSvvYAp4AoAJlrTUEeiM4y'
-				b'iaDrPOvgbXWkJcyFOc84XWaqqqqqrttyLM00wHZt4VhS6+4hdYEu47ygS8tayJY25QJVHfWBMbGGNwJj'
-				b'jfVgTG1AlYUCWrCyhWNIGW9sA0WgGLZJbSS2wlQiyCkFgOegGdIsMILFFWqdGiGB2ltttttttttttttt'
-				b'ttttttttttttttttttttzZghtzY3/FDwFk6Bxk2Jeg3gaLbarg4MtwhS8t96OD+hMV7HZDGTA7+XoC8f'
-				b'l63mbZIQnB26xbge4qgVvDJMfZ9bIvUnApzGKAX1cuJgIQGZilijdnfXDQJjGgdSeWUCwsLoJBtffJkj'
-				b'Em3W4thAQh2ZWmlMCGhQ4PP6+vxtfia+XX0MJvg2osxiOOOOOOOOOOwxHUSdrTA3+xy76oB3Pml+fvMa'
-				b'7bASrNkjhK682MjIaeGbOpvbZA1GKJGzbtRxtetimRrP+BtbAAwOSq+11kAjJIwMPLuvH6P9mlBBXxev'
-				b'5HjY9f6+8Ee5O4Q7+jxu/rs/gbw9mSCef70drwr7Z3e21ATXFEF7prLOy4lu2qsZgkwG1NW2YcfIvzXO'
-				b'dcr5erPa7ogu8cRFubteztm1sqH52/9tZHxAIB0MNeFXAIhDWZu52HW8bXDLAOp6HMuedu7MwxMOVeBA'
-				b'XS/YL0Dl5iOTzTkgKQMDt9QUgVueEHtSQ+odnJVQKCZ1XGDc7WHFDBcWX8aeJr0a0ERhv1Wq7nFrgB28'
-				b'7/q8nddO/ttc2OS9oBhHgy3DYmF99M4U3nm2uxJYhGACT6nubJmAkkBL7dyTzxO2orMheG3LSoJS6dHW'
-				b'SbOnZQDlne72W3nef0+G/pObEBFv+pv2LoAIPSgKuU8T/Ry4XgATYQDoRQd6DtgxU6V5gAKh1u32B0pc'
-				b'd4+jjZQAhf+oqwF/hfg/g3CfVtRIN6GZc/RYc//cSjCnXTXM7wXGjs9jaTES88KS5k7rulq+f5mP6/A/'
-				b'nw/Q3mF1/f+zxcW/5f97nJ8jKtc3F8b2Op1db4etl62te0eTRwKTd09PHQUuEPF39HrPp6enp5ppppra'
-				b'bX/k4/fv379/wHk3sYEF7Rv+vbXG4se1YSdHwo2rVVVr61tTbnE7/Kp4OJ+23qKnpXnRq7TtQupY1MK8'
-				b'wLP32re8bs7+/vcC2vLfL+fmXmZz7fhXWN6GHdwru7/PFyr2+voUK/v+o8F4NEzqyIv4CtnqerW5ZyMf'
-				b'IYq7kcpTzvJ5VF1dd1mv8KWXQ7jJI2KFbKJnbn4X3XYboa/GuwjpTQOmWHAiEhAEAD6AwrdNKzLhpMmI'
-				b'H7U+p9vw3aUD8LKBLRQkUPjwBPfghaIhIIfHio1ECQSRH0Yo92AffRBdN9St/VciWiQFAehxVbrfR3eO'
-				b'/yJrx+AtonPOU2WIBF7l+5x06X8q6pN1iqHZAgjArqXdGxE3AaIHEl2JVfXZWNvuu39bLCK/MyGIyK+Q'
-				b'71H5mVu/w9/fzuvr4e7tcwuZqgplW5yXljr+xgtf9vAB7HM/7VY/TR2D00u102ZEMu0HXAkd3Zamfa9g'
-				b'sF6nK680fqYytriF+62E6SYyWqEOljTq5J9KQs5m0oeVNTPfGeL+1rZhn9L4USD5dOShJzSyRMBAVNKF'
-				b'7ZS7UsrHxuunHDvaMfM/Bt/1zTcp7Cvv8InupHlHWdjggK08ArPZFI/0Q5fDzMD+f3PVe8a6gdUMsyIp'
-				b'YnesuICROqlkH1sjw1mVCIy4X7uWGkuXTmXTzDYlMW1CAgJlnixV2t1WpYph7s/+N1vD/se0LLdS4Akg'
-				b'E60mDl1MBK0zadFNTE11yrTGwUAWsGAIKjw4qJhiinZxULHHqRBGpFPhxQ66ChIISILIoMiAzw6F9yCl'
-				b'oIaoBriyD2IMgdWKgc6IoMggpfFEO+iAaIItkQEkQF4EQOmigaib2K1RWRVYR7pYSsgsgsBQFwkrIpFg'
-				b'wJEkJBJAQ0QEBkBHNE0QJHRAk0fBQfj/vaQXTAVSREFMkAUM0AFkikiwihFkFILAUBZFJIkgSMhISLKF'
-				b'UDPpu1OBXtQcNqtKhZAo2SqhCSE1R+lahDa3/euLvl6bFlAkgyHuU/14vKQOnC9Jo2qEA9mXRZH5NUAd'
-				b'BCcomHQ1ICQKlWAskDWQ0s3OVnFz4JbkM/3k8/HADCCfcTTbRLQm9HrQlFB4fUrUzBhFuinHYpA4rjqg'
-				b'pcHYigXwSCEXdYCFCE5JzSqo6vzP5pfdee+Yfdzp+t4GBnhIMiLt+BQMGKSiCxGDGO643lfk5iQzfKpw'
-				b'Xoq2RJQQRdhydgJCCipGQEiidpP/i7kinChIEsAF0oA='
-			)
-		case _:
-			raise ValueError()
-	with open(file, 'wb') as f:
-		f.write(
-			bz2.decompress(
-				base64.decodebytes(
-					b64
-				)
-			)
-		)
+    match papi_version:
+        case '3.3.6':
+            b64 = (
+                b'QlpoOTFBWSZTWZodmZEAE8l/////////////////////////////////////////////4DCfNe706t2+'
```

Comment:
Consider extracting this bytes content to a dedicated file at some point, it may be easier for the formatting.